compiler/src/iree/compiler/Codegen/Common/GPU/GPUVectorDistribution.cpp

```diff
@@ -235,11 +235,11 @@ static void applyVectorDistribution(Operation *root,
   applicator.applyDefaultCostModel();
 
   // Collect all the operations to be distributed.
-  std::deque<Operation *> worklist;
+  llvm::SmallDenseSet<Operation *> worklist;
```

Comment:
I think vector distribution insensitive to work ordering. Am I wrong here? 

Why this here now ?
I had to do this because re-distribution listener will keep adding ops to work list even if we do a operand change. Then we end up with same operation in the worklist multiple times. If folks have a better solution, Im all ears :)

Reply:
found a better way: just remove the redistribute attr when it get scheduled.